src/previewFrame/index.js

```diff
@@ -7,57 +7,16 @@ import uniq from 'lodash/uniq';
 import loopBreaker from 'loop-breaker';
```

Comment:
I love that you’ve broken the preview generator into multiple submodules, but I think the whole thing should probably still live in `src/util` (e.g. `src/util/previewFrame/index` etc.). The general principle is that top-level directories inside `src` should group together architecture-level categories of code (reducers, sagas, components, etc.) rather than specific concerns. `util` is kind of the “junk drawer” for stuff that doesn’t fit neatly into a category with other extant modules (not to say the code in there is junk!)

Reply:
Moved to util